login_check.php

```diff
@@ -42,6 +42,6 @@
 } else {
     echo "you have to login to see this page";
 }
-header("Location: redirect.html");
-?>
+header("Location: redirect.php");

```

Comment:
Why does this redirect to redirect.php ... which just redirects to contact_list.php